src/lib/geolocation.ts

```diff
@@ -7,12 +7,16 @@ type CountryGroupId =
     | 'NZDCountries'
     | 'Canada';
 
+// Used to internationalise 'Support the Guardian' links
+export type SupportRegionId = 'UK' | 'US' | 'AU' | 'EU' | 'INT' | 'NZ' | 'CA';
```

Comment:
Really minor but I wonder if there's an argument to having these be lower case? Since we always lowercase when we add to the url. It also might make them look visually different to a country code, so easier to tell apart at a glance.

Reply:
Interesting that you noticed that (in a good way)! This was mostly ported from Frontend and they do use lowercase here (and then `toLowerCase()` again when adding to the URL, for some reason) but I wanted all of these country codes and region IDs to have the same capitalisation and abstract any transformations into the helper functions that the rest of the app interfaces with. I don't have a strong view either way, but my thinking was that having this particular type of internationalisation code use a different capitalisation from the others would only raise more questions.